src/pages/debrief/cat-a-mod1/__tests__/debrief.cat-a-mod1.page.spec.ts

```diff
@@ -38,8 +38,7 @@ import { SpeedCheckDebriefCardComponent } from '../components/speed-check-debrie
 import { SetSingleFaultCompetencyOutcome }
   from '../../../../modules/tests/test-data/cat-a-mod1/single-fault-competencies/single-fault-competencies.actions';
 import { CompetencyOutcome } from '../../../../shared/models/competency-outcome';
-import { singleFaultCompetencyLables }
```

Comment:
I know the reference is removed - but the misspelling on singleFaultcompetencyLabels is annoying me :)